src/twenty-three-video/twenty-three-video.tsx

```diff
@@ -12,44 +16,42 @@ import GlobalStyles from "../GlobalStyles";
  * @param {Function} props.slideDone Function to invoke when the slide is done playing.
  * @returns {object} The component.
  */
-function TwentyThreeVideo({ slide, content = {}, run, slideDone }) {
+const TwentyThreeVideo: FC<TwentyThreeVideoProps> = ({ slide, content, slideDone }) => {
   const iframeId = slide["@id"];
 
   // TODO: This has to be dynamic
   const videoUrl = "video.kk.dk";
 
   // Content from content
   const {
-    videoList = "",
     autoPlay = true,
     showTray = false,
     mutedAutoPlay = true,
     autoMute = false,
+    jsonData
   } = content;
 
   // change boolean to number
   const convertedAutoplay = autoPlay === true ? "1" : "0";
   const convertedShowtray = showTray === true ? "1" : "0";
   const convertedMutedAutoPlay = mutedAutoPlay === true ? "1" : "0";
   const convertedAutoMute = autoMute === true ? "1" : "0";
-
-  // This fix normal typing errors and cleans the array for empty items
-  const formattedVideoList = videoList
-    .replaceAll(".", ",")
-    .split(",")
-    .map((item) => item.trim())
-    .filter(Boolean);
+  let videoList: any[] = [];
+  try {
+    videoList = JSON.parse(jsonData);
+  } catch(e) {
+    slideDone(slide);
+  }
 
   const [currentVideoIndex, setCurrentVideoIndex] = useState(0);
-  const [videoId, setVideoId] = useState(formattedVideoList[currentVideoIndex]);
 
   /** @param e */
```

Comment:
That docblock could use an update.

Reply:
Yes, if only original author had provided a more meaningful docblock, that would have been possible :)